robottelo/ui/rhai.py

```diff
@@ -28,14 +29,18 @@ def register_client_to_rhai(self, activation_key, org):
 
         # Red Hat Access Insights requires RHEL 6/7 repo and it not
         # possible to sync the repo during the tests, Adding repo file.
-        self.vm.configure_rhel_repo(conf.properties['clients.rhel6_repo'])
-
-        try:
+        if rhel_distro == 'rhel67':
+            rhel_repo = conf.properties['clients.rhel6_repo']
```

Comment:
The configuration is definitely required for testing as we would always require to test the latest rpm. Added the try and except block again for handling the optional part. thanks.